cql2pgjson/src/test/java/org/folio/cql2pgjson/CQL2PgJSONTest.java

```diff
@@ -167,6 +168,7 @@ public void cql2pgJsonException(CQL2PgJSON cql2pgJson, String cql,
 
   @Test
   @Parameters({
+    "\"field'))@@to_tsquery(('english\"=x # ",
```

Comment:
This is the basic(String) method, but this is not a basic test. Please move it into some other test method and add documentation if needed so that it is clear what class of input this tests (SQL injection in field name?)

Reply:
Sure . Can make a separate test for that.